src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationOptions.cs

```diff
@@ -252,13 +252,9 @@ public ISecureDataFormat<AuthenticationProperties> StateDataFormat
             {
                 return _stateDataFormat;
             }
+            [param: NotNull]
```

Comment:
@PinpointTownes what I have seen here https://github.com/aspnet/Common/blob/dev/src/Microsoft.Framework.NotNullAttribute.Sources/NotNullAttribute.cs#L8 is ```[NotNull]``` usage is ```Parameter``` not ```Property```